test/integration/services/patient-quick-search.js

```diff
@@ -393,7 +393,7 @@ context('Patient Quick Search', function() {
       body: {
         data: [{
           ...searchResult,
-          attributes: { ...searchResult.attributes, match: { label: 'DOB', value: '2008-01-16' } },
```

Comment:
`label: 'DOB'` was incorrect. Should have been `label: 'Birth Date'`. But I ended up just switching this to use the patient name as the search match value.

And then below I added a couple tests to ensure the UI was re-rendered on the copy/paste. By verifying the results and label column are emptied.